packages/apputils-extension/schema/notification.json

```diff
@@ -25,11 +25,23 @@
   },
   "additionalProperties": false,
   "properties": {
+    "checkForUpdates": {
+      "title": "Check for JupyterLab updates",
+      "description": "Whether to check for newer version of JupyterLab or not. If `true`, it will make a request to a website.",
```

Comment:
```suggestion
      "description": "Whether to check for newer version of JupyterLab or not. It requires `fechNews` to be `true` to be active. If `true`, it will make a request to a website.",
```